airflow/models/dag.py

```diff
@@ -305,8 +305,9 @@ class DAG(LoggingMixin):
         characters, dashes, dots and underscores (all ASCII)
     :param description: The description for the DAG to e.g. be shown on the webserver
     :param schedule: Defines the rules according to which DAG runs are scheduled. Can
-        accept cron string, timedelta object, Timetable, or list of Dataset objects.
-        See also :doc:`/howto/timetable`.
+        accept cron string, timedelta object, Timetable, or list of Dataset objects. 
+        If no schedule (or schedule_interval) is provided, then schedule will be set 
+        to  `timedelta(days=1)`. See also :doc:`/howto/timetable`. 
```

Comment:
```suggestion
        If this is not provided, the DAG will be set to the default
        schedule ``timedelta(days=1)``. See also :doc:`/howto/timetable`. 
```

Let’s not mention the deprecated argument.

Reply:
@uranusjr, I thought about that but I'm order for the default to be used, both have to be NOTSET